src/styles/blocks/page.scss

```
@@ -0,0 +1,29 @@
+.page {
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: normal;
+  font-size: 10px;
+  min-width: 1024px;
+
+  &__section {
+    display: flex;
+    align-items: flex-start;
+    flex-wrap: wrap;
+    box-sizing: border-box;
+    max-width: 1024px;
+    margin: 0 auto;
+    padding: 50px 40px;
+  }
+
+  &__product-item {
+    margin-right: $horizontal-gap;
+    margin-bottom: $vertical-gap;
+  }
+
+  &__product-item:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__product-item:nth-last-child(-n + 4) {
+    margin-top: 0;
+  }
+}
```

9-st point of the CHECKLIST 👉 [checklist.md](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)
```suggestion
  &__product-item {
    margin-right: $horizontal-gap;
    margin-bottom: $vertical-gap;
    
    &:nth-child(4n) {
    margin-right: 0;
  	}

  	&:nth-last-child(-n + 4) {
    margin-top: 0;
  	}
  }
}
```